test/lib/views/eventloop-view.spec.js

```diff
@@ -1,7 +1,10 @@
 "use strict";
 
-var expect = require("chai").expect;
+var chai = require("chai");
 var sinon = require("sinon");
+var sinonChai = require("sinon-chai");
+var expect = chai.expect;
+chai.use(sinonChai);
```

Comment:
We can work through it on Monday. If we've got you Slack access, just hit me or @jasonwilson up and we'll work it out.

And generally, speaking, anytime you're doing a repeated, fundamental change to a bunch of files, feel free to bounce that off of us early so we can possibly help debug things up front and save you some time!